client/src/api/graduates.js

```diff
@@ -0,0 +1,33 @@
+export const graduates = () =>{
+	let grdats= {
+		"OBakir90":false,
+		"csfilipinyi":true,
+		"Buchraatkeh1984":true,
+        "FarhanaFKhan":false,
+        "juditlehoczki":false
+    };
```

Comment:
Something's gone wrong with the indentation here.
```suggestion
	let grdats= {
		"OBakir90":false,
		"csfilipinyi":true,
		"Buchraatkeh1984":true,
                "FarhanaFKhan":false,
                "juditlehoczki":false
        };
```